src/reducers/statusReducer.js

```diff
@@ -23,7 +23,8 @@ export const initialStatusState = {
   message: '',
   timeoutID: null,
   file: null,
```

Comment:
`state.file` doesn't seem to be in use anymore in favor of using a local useState to hold `file`. Going to remove this and `CLEAR_FILE`, like what was done with `SET_FILE`.

Reply:
These are all good things to note, but we can address them in the next deploy. Can you file some issues, and we can follow up on them later.

Reply:
Let's focus on getting the current set of features out to the public

Reply:
Making note that we should remove CLEAR_FILE and related lines in reducer, reducer state, and typedefs. But could repurpose `StatusNotification` to use `file.name` from the local useState for `StatusMessage`